src/index.ts

```diff
@@ -11,28 +11,28 @@
  * See the `examples/` for full examples of how to use the HTTP service.
  * ```typescript
  * import { createLibp2p } from 'libp2p'
- * import { http } from '../dist/src/index.js'
- 
+ * import { http } from '@libp2p/whatwg-fetch'
+ *
  * async function main () {
  * const node = await createLibp2p({
  *     // other options ...
  *     services: {
  *       http: http()
  *     }
  * })
- 
```

Comment:
Was this why docs failed to generate for me?

Reply:
No, it's because the package.json description field wasn't found in the README anywhere - basically [this bit](https://github.com/ipfs/aegir/blob/master/src/docs/readme-updater-plugin.js#L207-L215).

It could use printing an error message if it can't be found, tbh..